src/tests/tracing/eventpipe/simpleruntimeeventvalidation/simpleruntimeeventvalidation.cs

```diff
@@ -64,6 +64,9 @@ public static int TestEntryPoint()
                 RuntimeEventValidation eventValidation = new RuntimeEventValidation();
                 eventValidation = null;
                 GC.Collect();
+
+                // Explicitly targeting AllocationTick event
+                GC.KeepAlive(new int[1000]);
```

Comment:
Does the GC.KeepAlive do anything here? Is it just to make sure the compiler doesn't optimize it away completely?

Reply:
yes, just to make sure.